unstructured/partition/pdf.py

```diff
@@ -1067,12 +1067,13 @@ def check_annotations_within_element(
     """
     annotations_within_element = []
     for annotation in annotation_list:
-        if annotation["page_number"] == page_number and (
-            calculate_intersection_area(element_bbox, annotation["bbox"])
-            / calculate_bbox_area(annotation["bbox"])
-            > threshold
-        ):
-            annotations_within_element.append(annotation)
+        if annotation["page_number"] == page_number:
+            annotation_bbox_size = calculate_bbox_area(annotation["bbox"])
+            if annotation_bbox_size and (
```

Comment:
From the error, I feel like `None` comes from `word` rather than `annot`. Do you think it makes sense to catch `NoneType` at `map_bbox_and_index` as well?